uint32 part_id
float32 x
float32 y
float32 confidence